geodirectory-functions/map-functions/map_functions.php

```diff
@@ -55,12 +90,34 @@ function send_marker_jason_to_js()
                     $canvas_jason = '[{"totalcount":"0"}]';
 
                 $map_canvas_jason_args = array($canvas . '_jason' => $canvas_jason);
+
+                /**
+                 * Filter for map canvas json args
+                 *
+                 * You can use this filter to modify map canvas json args.
+                 *
+                 * @since 1.0.0
+                 * @package GeoDirectory
+                 * @param string $canvas map canvas array key.
+                 * @param array $map_canvas_jason_args map canvas args.

```

Comment:
when a filter is inside a function please add the @see tag to the filter for the main function
@see send_marker_jason_to_js function.